### PR TITLE
feat(crd): allow adding custom init containers to dragonfly sts

### LINKED ISSUE #170: Support initContainers
My instance (volumes) can only be written by UID 0 (root) normally unless i change the ownership of the files/directories that are mounted using initContainers before the main containers start.
Or support changing the UID/GID (`runAsUser`, `runAsGroup`) but this is not recommended way.

### PR DESCRIPTION
**Pull request by gdsoumya**
Fixes #170

### CODE REVIEW

File: manifests/crd.yaml
Comment (Abhra303) [inline manifests/crd.yaml:1]: Can you drop manifest file changes please? I will update it before publishing a new version.

File: manifests/dragonfly-operator.yaml
Comment (Abhra303) [inline manifests/dragonfly-operator.yaml:1]: Can you drop the changes please?

File: manifests/crd.yaml
Comment (gdsoumya) [reply manifests/crd.yaml:1]: update

File: manifests/dragonfly-operator.yaml
Comment (gdsoumya) [reply manifests/dragonfly-operator.yaml:1]: update

**Review by Abhra303 — APPROVED** [review]

**Comment by Abhra303** [comment]
Yep, I will include it in the next release

**Pull request merged**